packages/webdriver/src/driver/index.ts

```diff
@@ -72,6 +72,7 @@ export async function startWebDriver (options: Options.WebDriver) {
         /**
          * Chrome
          */
+        caps.browserName = 'chrome'
```

Comment:
Let's not tweak the name, just make it lowercase
```suggestion
        caps.browserName = caps.browserName.toLowerCase()
```